### PR TITLE
refactor: Cleanup storybook components

### PR DESCRIPTION
**Pull request by JeangelLF**
## Ticket

https://github.com/corona-school/project-user/issues/1146

## What was done?

- Removed a bunch of unused components
- Fixed some stories that were throwing errors
- Removed some unneeded stories. For these, I tried to move the comments from the Story to the component file to not lose that documentation

I also wanted to upgrade Storybook to the latest version but I'll tackle that one in a separate task

### CODE REVIEW

File: src/widgets/ListItem.tsx [hunk 1]
```diff
@@ -1,29 +1,22 @@
...
 
 type Props = {
     label: string;
-    value?: string;
     onPress?: () => any;
     isDisabled?: boolean;
 };
 
-const EditDataRow: React.FC<Props> = ({ label, value, onPress, isDisabled }) => {
+const EditDataRow: React.FC<Props> = ({ label, onPress, isDisabled }) => {
```
Comment (Jonasdoubleyou) [inline src/widgets/ListItem.tsx:9]: const ListItem

File: src/components/CSSWrapper.tsx [hunk 2]
```diff
@@ -6,6 +6,10 @@ type Props = {
     style?: Object;
 };
 
+/**
+ * This wrapper component is used to apply css in places where native it not applicable or hard to achieve.
+ * There is a *.native file that was a fallback for natives contexts
+ */
```
Comment (Jonasdoubleyou) [inline src/components/CSSWrapper.tsx:12]: (I guess sooner or later we should finally decide whether we go all-in PWA or try native iOS / Android, and then either drop this component or rework all the other places where we have <div>s and the alike. Currently im more in favor of PWA)

File: src/components/AsNavigationItem.tsx [hunk 3]
```diff
@@ -6,6 +6,9 @@ type Props = {
     children: ReactNode | ReactNode[];
 };
 
+/**
+ * It allows to control the active navigation path by using a effect to set `rootPath`on initial rendering
+ */
```
Comment (Jonasdoubleyou) [inline src/components/AsNavigationItem.tsx:11]: (hm, not sure if we really need a global state just to highlight the icon in the navigation bar. Maybe we can just move the navItems into the bar, and then just decide based on the current location which one to highlight)

**Review by Jonasdoubleyou — APPROVED** [review]
nice cleanup!

File: src/components/CSSWrapper.tsx (hunk 2 above)
Comment (JeangelLF) [reply src/components/CSSWrapper.tsx:12]: Sounds good, I also think all-in PWA would be a nice idea.

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**